### ISSUE TITLE
I want to disable particular day for eg: Mondays, or Weekends

### ISSUE THREAD

**Issue by ranjanadangolcsu** [BugiDev/react-native-calendar-strip#156]
Is there any way I can disable a particular day in a week instead of dates or both?

**Comment by peacechen** [comment]
If you mean disable Monday every week, for example, no unfortunately that feature doesn't exist.  You could implement it by using the `datesBlacklist` prop, but you would need to specify each and every day that you want disabled.

PRs are welcome.  I suggest adding a prop such as `enableDates` or `disableDates` that accepts a function.  The user would supply a function that returns true/false for every date that's called.  In your case, check for the day of the week with `date.weekday() === 1` (Monday).
https://momentjs.com/docs/#/get-set/weekday/

Example:
```
<CalendarStrip
  enableDates={myEnableDatesFunc}
/>

function myEnableDatesFunc(date) {
  // Return true for all days that aren't Monday
  return date.weekday() !== 1;
}
```

**Comment by mikefriday91** [comment]
`useIsoWeekday={false}`
> start week on ISO day of week (default true). If false, starts week on startingDate parameter.

**Comment by peacechen** [comment]
This feature has been released in 1.4.2. See the Readme for an example passing a callback to `datesBlacklist`.
